lib/strings.md

```diff
@@ -5,20 +5,30 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A string is a line of text or values typically created by using quotations.
 
 # What are some examples of information that would be Strings as opposed to some other data type?
 
-Replace me with your response...
+Strings are created by enclosing values with quotations and by doing such any 
+mathematical operator used on a string will return a string value instead of an 
+integer value like some other data types would.
 
 # What is one way, using Ruby, to retrieve the 6th character in a String like `"Ada Lovelace"`? How about the 8th character? What happens if you try to retrieve the value of the _99th_ character (Or any character that doesn't exist)?
 
-Replace me with your response...
+Given a string of a specific length, using a code to call for a character within
+the string should be performed by defining the string as a variable ex. a_string = "string"
+and then performing the ruby command to retrieve the nth value of a_string. Given 
+that a string has n length, where the first character is the zero-ith position, 
+a request for the n+1 position will return a nil response.
 
 # The previous question asks about finding, for example, the 6th character in a String. Is it possible to find the **-6th** (Notice the negative symbol!) character in a String? What does that even mean?
 
-Replace me with your response...
+A request for a character with a positive number will start at the beginning of 
+the string and count its way til the end of the string. Using a negative number 
+will start the count at the end of the string and work its way from right to left.
 
 # What is one way, using Ruby, to replace certain characters in a string with some other set of characters? For example, given `"Sumeet Jain"`, how would you replace all of the `e` characters in my name with exclamation marks? (So it would be `"Sum!!t Jain"`.)
 
-Replace me with your response...
+The sub command in the rubymonk link is used by first defining the a set of characters
```

Comment:
What is the difference between `sub` and `gsub`? Which is appropriate to use in this scenario? 

[This site](http://ruby-doc.org/core-2.2.0/String.html) may be helpful in answering.